packages/table/src/cell/cell.tsx

```diff
@@ -102,7 +102,17 @@ export class Cell extends React.Component<ICellProps, {}> {
             },
         );
 
-        const content = <div className={textClasses}>{this.props.children}</div>;
+        // add width and height to the children, for use in shouldComponentUpdate in truncatedFormat
+        // note: these aren't actually used by truncated format, just in shouldComponentUpdate
+        const modifiedChildren = React.Children.map(this.props.children, (child) => {
+            if (style != null && React.isValidElement(child)) {
+                return React.cloneElement(child as React.ReactElement<any>,
+                    {parentCellHeight: style.height, parentCellWidth: style.width});
```

Comment:
nit: proper format would look something like this. but that's why we have `prettier` incoming.

```tsx
return React.cloneElement(child as React.ReactElement<any>, {
    parentCellHeight: style.height,
    parentCellWidth: style.width,
});
```